src/gnatss/loaders.py

```diff
@@ -70,6 +69,7 @@ def load_sound_speed(sv_file: str) -> pd.DataFrame:
 def load_travel_times(
     files: List[str],
     transponder_ids: List[str],
+    is_j2k: bool = False,
```

Comment:
Something to consider is using an enum instead of a bool, e.g.
```
class DateFormat(Enum):
    J2000
    Default # or DateAndTime or something
```
This allows for more flexibility later on to handle other cases without changing the interface. If other cases are highly unlikely then the flag is fine.

Reply:
It's highly unlikely to be other than J2000 epoch since they're currently limited to the upstream proprietary software.